ui/app/helpers/utils/util.js

```diff
@@ -309,3 +309,27 @@ export function getAccountByAddress (accounts = [], targetAddress) {
 export function stripHttpSchemes (urlString) {
   return urlString.replace(/^https?:\/\//u, '')
 }
+
+/**
+ * Checks whether a URL-like value (object or string) is an extension URL.
+ *
+ * @param {string | URL | object} urlLike - The URL-like value to test.
```

Comment:
`activeTab` is not a string, but an object with a `protocol` property, much like a `URL` object.

Reply:
Of course, knowing that, we could simply pass it the `protocol` string of that object. I'm fine with that if that's the preference.

Reply:
Or the `origin`, which is what we used to be comparing against.

It doesn't really matter. Just that it's nice to avoid branches if possible, and it's nice to have clearer type signatures.